db/migrate/20170217095802_delete_problematic_translations.rb

```diff
@@ -0,0 +1,20 @@
+class DeleteProblematicTranslations < ActiveRecord::Migration[5.0]
+  def change
+  	require_relative '/var/apps/publishing-api/db/migrate/helpers/delete_content.rb'
```

Comment:
this shouldn't need the full absolute path and is more conventional at the top. See this [old one](https://github.com/alphagov/publishing-api/blob/319005711dd14a7f045895540a04f0cc6acf75d8/db/migrate/20160831103251_delete_duplicate_about_pages.rb) as an example